.github/workflows/silabs-firmware-build.yaml

```
@@ -49,7 +49,7 @@ jobs:
     name: Build firmware
     runs-on: ubuntu-latest
     container:
-      image: ghcr.io/nabucasa/silabs-firmware-builder:${{ inputs.sdk_version }}
+      image: ghcr.io/${{ github.repository }}:${{ inputs.sdk_version }}
```

Nit: `silabs-firmware-builder` is the image name. It happens to be the same as the repository name here but it doesn't need to be. 

```suggestion
      image: ghcr.io/${{ github.repository_owner }}/silabs-firmware-builder:${{ inputs.sdk_version }}
```

---

I actually realised after making this PR that can call this workflow remotely something like

```uses: NabuCasa/silabs-firmware-builder/.github/workflows/silabs-firmware-build.yaml@<commit or tag>```

In this case repository_owner is probably not what would be wanted, perhaps I will make this an input with default value instead.

Related to this I also noticed that the NabuCasa silabs image is not public, is that intentional?